bitsight/operations.py

```diff
@@ -0,0 +1,162 @@
+"""
+Copyright start
+MIT License
+Copyright (c) 2024 Fortinet Inc
+Copyright end
+"""
+
+from datetime import datetime
+
+from requests import request, exceptions as req_exceptions
+from connectors.core.connector import get_logger, ConnectorError
+from .constants import *
+
+
+logger = get_logger("bitsight")
+
+
+class Bitsight:
+    def __init__(self, config, *args, **kwargs):
+        server_url = config.get("server_url")
+        if not server_url.startswith('https://') and not server_url.startswith('http://'):
+            server_url = "https://"+server_url
+        self.url = server_url.strip("/")
+        self.username = str(config.get("api_token"))
+        self.verify_ssl = config.get("verify_ssl")
+
+    def api_request(self, method, endpoint, params={}):
```

Comment:
Data/JSON data is missing here, Even though no operation right now is using that, In future it might be required. Please add these params here